submissions/adeyemiJuwon/HTMLforms/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>marthas living brand</title>
+</head>
+<body>
+<header>
+    <div>
+        <input type="search" name="search" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="login.html">Log in</a>
+                </li>
+                <li>
+                    <a href="account.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="contact.html">Contact</a>
+                </li>
+                <li>
+                    <a href="#">Cart</a>
+                </li>
+            </ul>
+    </div>
+    <div>
+        <a href="index.html
+            <img src="img/mlogo.png" alt="">
+        </a>
+    </div>
+    <hr>
+    <nav>
+        <ul>
+            <li><a href="#">Best Sellers</a></li>
+            <li><a href="#">Baby Care &amp; Wear</a></li>
+            <li><a href="#">Women collection</a></li>
+            <li><a href="#">Electronic Accessiories</a></li>
+            <li><a href="#"> Lights &amp; Lighting</a></li>
+            <li><a href="#">Smart Home &amp; Garden</a></li>
+
+        </ul>
+    </nav>
+</header>
+    <hr>
+<section>
+    <div>
+        <h1>LAPTOP ACCESSORIES</h1>
+        <a href="">
+            <img src="img/laptop.jpg" alt="">
```

Comment:
Remove the images added to `html forms` @Samju200

Reply:
If i remove the image it will remain only the product description